cost/low_account_usage/low_account_usage.pt

```diff
@@ -33,7 +33,6 @@ parameter "param_billing_centers" do
   label "Billing Center Name"
   description "List of Billing Center Names to check"
   type "list"
-  min_length 1
```

Comment:
instead of min_lenth, we should just set the default to []. The description should also explain that the parameter should be left empty to run the policy against all Billing Centers.

For example:

parameter "param_billing_centers" do
  label "Billing Center Name"
  description "List of Billing Center Names to check. Leave blank to run policy against all Billing Centers"
  type "list"
  default []
end

Reply:
Addressed!